README.md

```diff
@@ -1 +1,112 @@
-# Busify
\ No newline at end of file
+<p align-self="center"><img src="./assets/images/Logo.gif" alt="Alt Text"></p>
+
+# Busify-Ticket Booking System
+
+A centralised bus ticket booking system that will allow the students to book tickets in advance through an online website, saving time and effort. Students will no longer need to physically stand in long queues to purchase tickets.
+
+The ticket booking system will provide a user-friendly interface where passengers can search for available routes, select their desired travel date and time and make secure online payments.
+
+# Features
+
+The proposed plan is to develop a bus ticket booking system consisting of the
+
+- Client App
+- Main Server
+- Notification Microservice,
+- Admin Panel, and
+- Mobile App for Conductors.
+
+### The Client App allows users
```

Comment:
Do not specify each sections here. We are going to have separate readme for each directory. 
For example, the frontend has its own readme under `/frontend`.